packages/tools/src/bundlers.ts

```diff
@@ -211,6 +214,18 @@ export const configXpack = (
         },
         plugins: config.plugins,
     };
+
+    // Add TypeScript support for webpack/rspack
+    if (hasTypeScriptEntries) {
+        baseConfig.resolve = {
+            extensions: ['.tsx', '.ts', '.js'],
+        };
+        baseConfig.module = {
+            rules: [{ test: /\.([cm]?ts|tsx)$/, loader: 'ts-loader' }],
+        };
+    }
```

Comment:
Does this even need to be conditional?
Don't we always want to use `ts-loader` for any `ts` file we encounter? I don't thing we need `hasTypeScriptEntries`.